src/Components/Components/src/PublicAPI.Unshipped.txt

```diff
@@ -8,6 +8,11 @@ Microsoft.AspNetCore.Components.ComponentApplicationState.PersistAsJson<TValue>(
 Microsoft.AspNetCore.Components.ComponentApplicationState.PersistState(string! key, byte[]! value) -> void
 Microsoft.AspNetCore.Components.ComponentApplicationState.TryTakeAsJson<TValue>(string! key, out TValue? instance) -> bool
 Microsoft.AspNetCore.Components.ComponentApplicationState.TryTakePersistedState(string! key, out byte[]? value) -> bool
+Microsoft.AspNetCore.Components.HotReload.HotReloadContext
+Microsoft.AspNetCore.Components.HotReload.HotReloadContext.HotReloadContext() -> void
+Microsoft.AspNetCore.Components.HotReload.HotReloadContext.IsHotReloading.get -> bool
+Microsoft.AspNetCore.Components.HotReload.IReceiveHotReloadContext
```

Comment:
Can we replace this with a property on `RenderHandle`? e.g. `RenderHandle.IsHotReloading`